docs/PowerAuth-Server-0.18.0.md

```diff
@@ -90,7 +90,7 @@ We added row level DB locking for activations when handling concurrent signature
 
 ```properties
 # Database Lock Timeout Configuration
-spring.jpa.properties.lock.timeout=10000
+spring.jpa.properties.javax.persistence.lock.timeout=10000
```

Comment:
What about keeping the wrong key in the old migration guide and mentioning it in `docs/PowerAuth-Server-1.5.0.md`?

Reply:
I think we do not need this value in new migration guide, it's not being changed by any client. We only mentioned it when introducing the value.